src/lib/integration_test_lib/util.ml

```diff
@@ -9,6 +9,61 @@ let run_cmd dir prog args =
   Process.create_exn ~working_dir:dir ~prog ~args ()
   >>= Process.collect_output_and_wait
 
+let drop_outer_quotes s =
+  let open String in
```

Comment:
Why is this needed?

Reply:
Because of the way `Yojson.Safe` serializes strings,
```
`String "hello"
```
is serialized to `"hello"`